.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: rust
```

Comment:
This file can be removed as Codecov action is not in `.github/workflows/codecode.yml`